@here/harp-mapview/lib/MapView.ts

```diff
@@ -789,6 +789,26 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_visibleTiles.getDataSourceCacheSize();
     }
 
+    /**
+     * Checks if cache size should be in MB (mega bytes). If `false` the size is in number of
+     * tiles. The default is `true`.
+     *
+     * @returns {boolean} `true` if the size is in MB, `false` otherwise.
+     */
+    getCacheSizeInMb(): boolean {
```

Comment:
To me it reads 'get cache size' - so like count bytes in cache.

Maybe, `{get,set)ResourceUsageMode(mode: 'basic' | 'memory'`) ?

Reply:
Right, bad name. Modifying it.